util/opentelemetry-util-genai/tests/test_fsspec_upload.py

```diff
@@ -417,3 +443,26 @@ def test_upload_jsonlines(self) -> None:
 {"role":"user","parts":[{"response":{"capital":"Paris"},"id":"get_capital_0","type":"tool_call_response"}],"index":2}
 """,
         )
+
+    def test_upload_chained_filesystem_ref(self) -> None:
+        """Using a chained filesystem like simplecache should refer to the final remote destination"""
+        hook = FsspecUploadCompletionHook(
+            base_path="simplecache::memory",
```

Comment:
This would previously record `simplecache::gs://mybucket/<uuid>-input.json` and now it just references the destination `gs://mybucket/<uuid>-input.json`